native/lambda_game_engine/src/game.rs

```diff
@@ -562,25 +566,19 @@ fn apply_zone_effects(
 fn nearest_player(
     players: &Vec<&mut Player>,
     position: &Position,
+    auto_aim_max_distance: f32,
```

Comment:
I think we can reuse the function for other purposes so `max_search_distance` or `max_lookup_distance` might be a better name

```suggestion
    max_search_distance: f32,
```